gcc/simplify-rtx.c

```diff
@@ -4013,6 +4022,10 @@ simplify_const_binary_operation (enum rtx_code code, machine_mode mode,
 	  const REAL_VALUE_TYPE *opr0, *opr1;
 	  bool inexact;
 
+	  /* Subnormals are not handled corretly with -fftz-math.  */
```

Comment:
Typo 'corretly'. Also maybe it's enough to add the comment once, for the first instance of the check. Maybe it could be possible to unify the check also to the beginning of the function somehow to avoid repeated code?